packages/editor/src/components/post-title/index.js

```diff
@@ -119,6 +120,13 @@ class PostTitle extends Component {
 								onFocus={ this.onSelect }
 								onKeyDown={ this.onKeyDown }
 								onKeyPress={ this.onUnselect }
+								/*
+									Only autofocus the title when the post is entirely empty.
+									This should only happen for a new post, which means we
+									focus the title on new post so the author can start typing
+									right away, without needing to click anything.
+								*/
+								autofocus={ this.props.isPostEmpty ? 'autofocus' : undefined }
```

Comment:
Doesn't React warn about this usage, since we're assigning the attribute, not the DOM property `autoFocus` (capitalized "F")? With the latter, I'd also expect this to be passed as a boolean, i.e.

```
autoFocus={ this.props.isPostEmpty }
```

Reply:
I didn't see a warning, also I meant to assign the attribute… will React do some magic and assign a value to a DOM attribute if passed like that?! 😱

Reply:
Related: https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html

Notably:

![image](https://user-images.githubusercontent.com/1779930/45173639-cae62c00-b1d6-11e8-9d1b-ec1560f64421.png)

Reply:
I missed that one, cool!